src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -47,16 +47,19 @@ emit('toggle-navigation', {
 			:inert="!open || undefined"
 			@keydown.esc="handleEsc">
 			<div class="app-navigation__body" :class="{ 'app-navigation__body--no-list': !$scopedSlots.list }">
-				<!-- The main content of the navigation. If no list is passed to the #list slot, stretched vertically. -->
+				<!-- @slot For in-app search you can pass a `NcAppNavigationSearch` component as the slot content. -->
+				<slot name="search" />
+
+				<!-- @slot The main content of the navigation. If no list is passed to the #list slot, stretched vertically. -->
 				<slot />
```

Comment:
Both slots are in a common flex container, and 

> If no list is passed to the #list slot, stretched vertically

having this, is it expected behavior, or we expect users to not use body together with the search?

![scroll](https://github.com/user-attachments/assets/d151568a-1ace-4ac8-b4c5-f86a693d087e)

Reply:
Fixed this is not expected.